src/lib/Refiner.ml

```diff
@@ -353,6 +353,41 @@ struct
     S.IdElim (tmot, t_refl_case, tscrut), fib
 end
 
+module Path =
+struct
+  exception Todo1
+  exception Todo2
+  exception Todo3
+
+  (*
+    A type
+    x,x=0 |- a : A
+    x,x=1 |- b : A
+    ------------------------------------
+    path x.A a b type
+   *)
+  let formation (tac_tp : T.tp_tac) (tac0 : T.chk_tac) (tac1 : T.chk_tac) : T.tp_tac =
```

Comment:
This rule will be unnecessary, because the path type is totally defined by the equation `el(path(...))`; so what we will need instead is a typing rule for the path _code_, something of type `chk_tac -> chk_tac -> chk_tac` or something along those lines.

Reply:
per discussion the other day: got it